android/src/main/java/com/reactnativehmssdk/HmsModule.kt

```diff
@@ -388,4 +388,8 @@ class HmsModule(reactContext: ReactApplicationContext) : ReactContextBaseJavaMod
       })
     }
   }
+
+  private fun releaseAllAssets() {
```

Comment:
@samarthjain10  why is a stub function here?

Reply:
removed